README.md

```diff
@@ -57,28 +41,27 @@ echo 'deb [signed-by=3D9E81D3CA76CDCBE768C4B4DC6B4F8E60B8CF4C] https://deb.ln-as
 sudo apt update
 ```
 
-If you're setting up a dedicated full node, you may want to leave out `desktop` component in step 6.
+If you're setting up a dedicated full node, maybe [on dedicated hardware](docs/dedicated-hardware.md), you may want to leave out `desktop` component in step 6.
 
-I recommend reading short [docs on each application](docs/user-level.md#using-applications) you plan to use.
+You can now install the desired applications using e.g. `apt install btcpayserver`.
+Don't worry about the dependencies. :)
+Read short [docs on each application](docs/user-level.md#using-applications) you plan to use.
 
-Five things to know before you start using this
------------------------------------------------
+## Five things to know before you start using this
 
 * Do **not** attempt to configure anything - it will just work
-* Do understand that bitcoind and other services will run automatically
+* Do understand that `bitcoind` and other services will run automatically **innediately after installation and after each boot**.
 * **By default, the LND wallet is created automatically after installation and the seed is stored in /var/lib/lnd-system-mainnet/.seed.txt**
 * Do **not** attempt to configure anything - it will just work
 * Make sure you have at least 400 GB of free space
```

Comment:
what about pruned node?

Reply:
It's easier to leave it out for now as `lnd`, `btc-rpc-explorer` and `electrs` cause turning off pruning without loud warning. I hope to address it in UI and make CLI users read about the details.